src/pushi/base/mail.py

```
@@ -28,8 +28,411 @@
+
+            # builds the MIME message for the email using the plain text
+            # content type for simple text emails
+            mime = email.mime.text.MIMEText(body)
+            mime["Subject"] = subject
+            mime["From"] = smtp_sender
+            mime["To"] = target_email
+            contents = mime.as_string()
+
+            # creates the callback function that is going to be used when
+            # the SMTP connection is closed after sending the email
+            def on_close(connection=None):
```

Ruff (0.14.10)

Unused function argument: `connection`

(ARG001)